### PR TITLE
Changes: Delete url=${url} for properties. Class BaseTest chang field…

### PR DESCRIPTION
**Pull request by oleg-kholyavchuk**
… softAssertions to soft. Class Constant added a field URL. Interface HttpService chang int statusCode to expectedCode. Class HttpServiceImpl chang int statusCode to expectedCode and method sendGetOffsetAndLimit Response to ValidatableResponse. Class Props chang field properties to PROPERTIES and method getProperty. Class chang Supporting to RestHelper and getListTodo, addListTodos, removeListTodos. Class Specification chang jsonToString. Class TodoApiNegativeTest add description in the annotation ParameterizedTest. Class TodoApiPositiveTest chang method checkTaskByOffset.

ADD: Class Util

### CODE REVIEW

File: src/main/java/com/bhft/util/Props.java [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+            PROPERTIES.load(inputStream);
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
 
     public static String getProperty(String key) {
         String property = System.getProperty(key);
-        return property == null ? properties.getProperty(key) : property;
+        if (!(property == null)) {
```
Comment (NezhinskijVV) [inline src/main/java/com/bhft/util/Props.java]: property != null

Comment (NezhinskijVV) [reply src/main/java/com/bhft/util/Props.java]: Можно без фигурных скобочек и без else

File: src/main/java/com/bhft/util/Props.java [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.Properties;
 
 public class Props {
-    private static final Properties properties = new Properties();
+    private static final Properties PROPERTIES= new Properties();
```
Comment (NezhinskijVV) [inline src/main/java/com/bhft/util/Props.java]: ctrl alt l

File: src/main/java/com/bhft/constant/Constant.java [hunk 3]
```diff
@@ -1,6 +1,8 @@
 package com.bhft.constant;
 
+@SuppressWarnings("All")
```
Comment (NezhinskijVV) [inline src/main/java/com/bhft/constant/Constant.java]: уберем

File: src/main/java/com/bhft/specification/Specification.java [hunk 4]
```diff
@@ -1,15 +1,21 @@
...
+import com.fasterxml.jackson.databind.ObjectMapper;
 import io.restassured.builder.RequestSpecBuilder;
 import io.restassured.http.ContentType;
 import io.restassured.specification.RequestSpecification;
+import lombok.SneakyThrows;
 
-import static com.bhft.helper.Supporting.jsonToString;
+import static com.bhft.constant.Constant.URL;
 
+@SuppressWarnings("All")
```
Comment (NezhinskijVV) [inline src/main/java/com/bhft/specification/Specification.java]: уберем

File: src/main/java/com/bhft/util/Props.java (hunk 1 above)
Comment (oleg-kholyavchuk) [reply src/main/java/com/bhft/util/Props.java]: Исправил

File: src/main/java/com/bhft/util/Props.java (hunk 2 above)
Comment (oleg-kholyavchuk) [reply src/main/java/com/bhft/util/Props.java]: Исправил

File: src/main/java/com/bhft/constant/Constant.java (hunk 3 above)
Comment (oleg-kholyavchuk) [reply src/main/java/com/bhft/constant/Constant.java]: Убрал

File: src/main/java/com/bhft/specification/Specification.java (hunk 4 above)
Comment (oleg-kholyavchuk) [reply src/main/java/com/bhft/specification/Specification.java]: Убрал

File: pom.xml [hunk 5]
```diff
@@ -107,9 +119,27 @@
                 <version>2.22.2</version>
                 <configuration>
                     <argLine>
-                        --illegal-access=permit
+<!--                        &#45;&#45;illegal-access=permit-->
```
Comment (NezhinskijVV) [inline pom.xml]: уберем

File: README.md [hunk 6]
```diff
@@ -41,49 +41,103 @@
 # Нагрузочное тестирование
 
 Тестирование производилось при помощи инструмента Jmeter.
-Показатели нагрузки измерялись средствами стандартных отчетов Jmeter.
+Показатели нагрузки измерялись средствами отчетов Jmeter.
```
Comment (NezhinskijVV) [inline README.md]: помінять строку запуска

**Pull request merged**